bitnami/concourse/templates/worker/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.worker.enabled -}}
+kind: PodDisruptionBudget
+apiVersion: policy/v1beta1
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: worker
+    {{- if .Values.commonLabels }}
+      {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
+    {{- end }}
+spec:
+  minAvailable: {{ .Values.worker.minAvailable }}
```

Comment:
We would normally use something like worker.pdb.enabled and worker.pdb.minAvailable